rules/group_rules.yaml

```diff
@@ -19,6 +19,10 @@
       # Must use quotes if the condition starts with @ symbol.
     - member_email: "@{DOMAIN}"
     - member_email: "@gmail.com"
+    # Service Accounts
```

Comment:
Can we make this comment more clear and descriptive of what the `gserviceaccount.com` are?  Such as:
```
GCP service accounts
```